windows/internal/build_magma.bat

```diff
@@ -0,0 +1,103 @@
+@setlocal
+
+set MAGMA_VERSION=2.5.4
+
+REM set CUVER_NODOT=%~1
+set CUVER=%CUVER_NODOT:~0,-1%.%CUVER_NODOT:~-1,1%
+
+REM set CONFIG=Debug
+
+set CONFIG_LOWERCASE=%CONFIG:D=d%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:R=r%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:M=m%
+
+echo Building for configuration: %CONFIG_LOWERCASE%, %CUVER%
+
+mkdir magma_cuda%CUVER_NODOT%
+cd magma_cuda%CUVER_NODOT%
+
+REM set "PATH=C:\Program Files\CMake\bin\;C:\Program Files\7-Zip\;C:\curl-7.57.0-win64-mingw\bin\;%PATH%"
+
+set "PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%\libnvvp;%PATH%"
+set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUVER%
+set NVTOOLSEXT_PATH=C:\Program Files\NVIDIA Corporation\NvToolsExt
+
+:: First install MKL, which provides BLAS and LAPACK API
+:: Download and install from:
+:: curl https://s3.amazonaws.com/ossci-windows/w_mkl_2018.2.185.exe -k -O
+:: .\w_mkl_2018.2.185.exe
+:: Follow the installer steps and install MKL to default path
```

Comment:
I'm not sure the above steps are still necessary--I did not run them in the GHA.

Reply:
If they're not necessary, I will remove these comments to remove confusion.

Reply:
@peterjc123 Unsure if you'd know the answer to this, linking you in case you do

Reply:
They are not needed I think.